blog/2025-07-21-openhab-5-0-release.md

```diff
@@ -0,0 +1,408 @@
+This series can be used on any time-based chart with either an `oh-value-axis` or `oh-category-axis` as the y-axis.
+
+<p align="center"><img style="max-width: 70%; min-width: 330px;" src="/uploads/2025-07-21-openhab-5-0-release/oh-state-series.png"/></p>
+
+### Semantic Tags Maintenance & Treeview Picker
+
+openHAB 4.1 introduced the ability to extend the semantic model with custom tags through a YAML configuration file.
+In openHAB 5.0, we have added support for maintaining these custom tags through the UI.
+
+The developer tools now include a new _Semantic Tags_ section, where you can view, edit, and delete your custom tags.
+You can view tag relationships and structure at a glance, making it easier to understand the concept of the semantic model and how to use it effectively.
+It features both a treeview and a code tab, allowing you to switch between a visual representation and the underlying YAML code.
+
+<p align="center"><img style="max-width: 70%; min-width: 330px;" src="/uploads/2025-07-21-openhab-5-0-release/semantic-tags-maintenance.gif"/></p>
+
+In addition to the maintenance page, the semantics picker in the Item editor has been updated to use the treeview, making it easier to select the correct semantic tag for your Items by visualising the hierarchy of tags:
+
+<p align="center"><img style="max-width: 70%; min-width: 330px;" src="/uploads/2025-07-21-openhab-5-0-release/semantic-tags-picker.gif"/></p>
+
+Again, a big thank you to Mark Herwege ([@mherwege](https://github.com/mherwege))!
+
+### Additional Enhancements
+
+- **Widget Config:** The widget config popup has been redesigned to support live updates, dirty checking and be movable, so you can watch the changes in your page while editing the widget config.
+- **Specific Browser Titles:** The browser title now reflects the current page, making it easier to identify the contents of the openHAB tabs in your browser.
+- **Log Viewer Enhancements:** The log viewer has been enhanced with a history buffer, allowing you to scroll back through previous log entries, and a log details popup that provides more information about each log entry.
+  Additionally, a text mode has been introduced as alternative to the default table view, having the look and feel of the well-known [frontail](https://github.com/mthenw/frontail) log viewer.
+- **Thing Inbox:** Add search, approve multiple Things at once and allow to change Thing ID & label when approving Things.
+
+## Blockly Enhancements
+
+- New Rule-Enable Block to enable or disable rules.
+- New dictionary block to update dictionary entry values
+- Persistence Block: Riemann sum support added
+- Boolean-typed variable added
+- Get Item Previous State added
+
+## Sitemap & Basic UI Enhancements
+
+- New `interpolation` parameter for charts to change how the line between two data points is drawn: Either `linear` or `step`.
```

Comment:
Please remove this line, it looks like this sitemap feature is not yet implemented in any sitemap UI!

Reply:
Done.